enterprise-suite/templates/es-console.yaml

```
@@ -176,7 +176,7 @@ data:
       # bookmarks / reloads in the /workloads/ path
       # (for our single page app, these should reload index.html but keep the path)
 
-      location ~ (.*/)(workloads/.*) {
+      location ~ (.*/)(namespaces/.*) {
```

I wonder if this would break if the workspace itself has the word "namespaces" in it? It's pretty edge case I suppose. To fix, we could think about including `workloads` too?
```suggestion
      location ~ (.*/)(namespaces/[^/]+/workloads/.*) {
```

---

es_workload="namespaces"
namespace="workloads"

evil. That's going in the tests. :-)